packages/big-design/src/components/Worksheet/spec.tsx

```diff
@@ -866,4 +923,91 @@ describe('ModalEditor', () => {
 
     await waitForElement(() => screen.getAllByRole('combobox'));
   });
+
+  test('renders in disabled state', async () => {
+    render(<Worksheet columns={disabledColumns} items={items} onChange={handleChange} />);
+
+    const buttons = screen.queryAllByText('Edit');
```

Comment:
judging by the `buttons` name, i imagine this is a button? if so `getByRole('button', ...)` is likely a better choice here